s3/libirods_s3.cpp

```diff
@@ -137,6 +137,18 @@ int get_booleans_from_host_mode(const std::string& host_mode_str,
     return 0;
 }
 
+// gets the resource name from the property map
+std::string get_resource_name(irods::plugin_property_map& _prop_map) {
+
+    std::string resc_property_name_str = "";
```

Comment:
The assignment of empty string is unnecessary.

Reply:
Made the change.